packages/grant-explorer/.env.sample

```diff
@@ -31,3 +31,8 @@ REACT_APP_TAG_MANAGER=XXXXX
 
 #Grant API Endpoint
 REACT_APP_GRANTS_API_ENDPOINT=#########
+
+# Passport
+REACT_APP_PASSPORT_API_KEY=#########
+REACT_APP_PASSPORT_API_ENDPOINT=##########
+REACT_APP_PASSPORT_API_COMMUNITY_ID=###
```

Comment:
Could we add documentation on how one would go about getting these values in the README file

Reply:
Sure - I removed the `REACT_APP_PASSPORT_API_COMMUNITY_ID=###` because it doesnt make sense in the env. We need a way to fetch the community ID for the round / user.

Reply:
removed and hardcoded for now